server/base/src/main/java/org/apache/accumulo/server/constraints/MetadataConstraints.java

```diff
@@ -224,8 +225,15 @@ public List<Short> check(Environment env, Mutation mutation) {
         try {
           TabletHostingGoalUtil.fromValue(new Value(columnUpdate.getValue()));
         } catch (IllegalArgumentException e) {
-          violations = addViolation(violations, 4);
+          violations = addViolation(violations, 10);
         }
+      } else if (ServerColumnFamily.OPID_COLUMN.equals(columnFamily, columnQualifier)) {
```

Comment:
I think you need to update the check on line 207 that checks that the Value is not empty.